Sky Atlas/Sky Atlas.html

```diff
@@ -2712,10 +2720,11 @@ <h1>Sections & Primary Docs</h1>
           <td>Core</td>
           <td>The role of the Support Facilitators in the Standby Spell process is as follows:
             <ul>
-              <li><p>Standby Spells can only be used in response to a properly validated and categorized emergency scenario, for which the Support Facilitators are responsible. See <dfn>A.1.8 - Emergency Response System - Emergency Response - Incident Validation</dfn> and <dfn>A.1.8 - Emergency Response System - Emergency Response - Incident Categorization</dfn>.</p><p>The decision to use a Standby Spell is reserved for the Support Facilitators; where possible, the Support Facilitators should consult with the responsible Facilitator of the impacted Scope(s) and any relevant Scope Advisor(s). See <dfn>A.1.8 - Emergency Response System - Emergency Response - Facilitators’s Roles And Responsibilities</dfn>.</p></li>
+              <li>Standby Spells can only be used in response to a properly validated and categorized emergency scenario, for which the Support Facilitators are responsible. See <dfn>A.1.8 - Emergency Response System - Emergency Response - Incident Validation</dfn> and <dfn>A.1.8 - Emergency Response System - Emergency Response - Incident Categorization</dfn>.</li>
+              <li>The decision to initiate the process to use a Standby Spell is reserved for the Support Facilitators; where possible, the Support Facilitators should consult with the responsible Facilitator of the impacted Scope(s) and any relevant Scope Advisor(s). See <dfn>A.1.8 - Emergency Response System - Emergency Response - Facilitators' Roles And Responsibilities</dfn>.</li>
```

Comment:
This isn't two bullets, it's a single bullet split into paragraphs.

```suggestion
              <li><p>Standby Spells can only be used in response to a properly validated and categorized emergency scenario, for which the Support Facilitators are responsible. See <dfn>A.1.8 - Emergency Response System - Emergency Response - Incident Validation</dfn> and <dfn>A.1.8 - Emergency Response System - Emergency Response - Incident Categorization</dfn>.</p><p>The decision to initiate the process to use a Standby Spell is reserved for the Support Facilitators; where possible, the Support Facilitators should consult with the responsible Facilitator of the impacted Scope(s) and any relevant Scope Advisor(s). See <dfn>A.1.8 - Emergency Response System - Emergency Response - Facilitators' Roles And Responsibilities</dfn>.</p></li>
```